scripts/tooling/Dockerfile

```diff
@@ -21,7 +20,6 @@ RUN GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@${
 RUN GO111MODULE=on go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@${JB_VERSION}
 RUN go get github.com/brancz/gojsontoyaml
 RUN go get github.com/campoy/embedmd
-RUN GO111MODULE=on go get github.com/go-bindata/go-bindata/v3/go-bindata@${GO_BINDATA_VERSION}
```

Comment:
This might cause potential issues in other repositories depending on tooling image.

Reply:
Do we have an issue regarding the tooling migration where I could reference that?

Reply:
no, feel free to open one :)